src/magento2/.ci/sample-static/workspace.yml

```diff
@@ -19,6 +19,19 @@ attribute('app.repository'): null
 attribute('aws.id'): null
 attribute('aws.key'): null
 
+attribute('magento.static_content'):
+  backend:
+    languages:
+      - en_GB
+      - en_US
```

Comment:
I know we can't turn off stores in spryker harness, but with the inherent array merge issues should we solve it for new work?

Reply:
or @dcole-inviqa could we have help in working out a fix to workspace?

Reply:
Ah, re-read this comment. We define as empty in the harness to avoid this problem. This is for testing if it works in Jenkins with some defined settings. Dynamic one is deliberately empty to test that scenario too (which compiles all languages and themes).

Reply:
Have created https://github.com/my127/workspace/issues/64 to track this, as I know it's been bugging everyone